tools/ZydisInfo.c

```diff
@@ -1319,6 +1319,11 @@ int main(int argc, char** argv)
     ZYAN_PUTS("");
     PrintSegments(&instruction, &data[0]);
 
+    ZydisEncoderRequest request;
+    ZyanUSize len = instruction.length;
+    status = ZydisEncoderDecodedInstructionToEncoderRequest(&instruction, operands, instruction.operand_count_visible, &request);
+    status = ZydisEncoderEncodeInstruction(&request, data, &len);
+
```

Comment:
Leftover from debugging?

Reply:
Whoops. Yeah. We need a simple command line tool like `ZydisInfo` which does re-encoding to debug specific byte sequences.

Might not make super much sense, but maybe we can even add "size optimized" (re-encide) as a new info output to `ZydisInfo`.